frontend/app/mappers/benefit-renewal-service-mappers.server.ts

```diff
@@ -59,7 +59,7 @@ export interface ToBenefitRenewRequestFromRenewAdultChildStateArgs {
   dentalBenefits: DentalFederalBenefitsState & DentalProvincialTerritorialBenefitsState;
   dentalInsurance: boolean;
   partnerInformation: PartnerInformationState | undefined;
-  contactInformation: ContactInformationState;
+  contactInformation?: ContactInformationState;
```

Comment:
Shouldn't we expect `contactInformation` to be required when mapping to the renew request? 

The applicant should have answered `isNewOrUpdatedPhoneNumber` and `isNewOrUpdatedEmail` at the very least.

Reply:
right, I removed a few too many conditionals in the route-helper and over-corrected in the mapper.